src/pages/index.js

```diff
@@ -36,6 +36,7 @@ const IndexPage = ({
 
   return (
     <Layout>
+      <SEO title="All recipes" />
```

Comment:
What does ` SEO` do?

Reply:
file [here](https://github.com/byteme980/recipes-md/blob/main/src/components/seo.js). it sets the document.title along with some meta open graph tags